src/main/java/com/robosh/ejournal/entity/HomeTask.java

```diff
@@ -0,0 +1,27 @@
+package com.robosh.ejournal.entity;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import java.time.LocalDate;
+
+@Data
+@NoArgsConstructor
+@Builder
+@Entity
```

Comment:
I think we need to put  argument in @Entity(name="home_task") in order if we will want to change class name. Need to be discussed with all team members.